Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <title>Test2</title>\r\n</head>\r\n<body>\r\n    <header>\r\n\r\n\r\n        <div class=\"oui\">\r\n            <h1>Oui il fait beau</h1>\r\n            <h2>Mais des fois il fait moche</h2>\r\n            <p>Creer une branche</p>\r\n        </div>\r\n\r\n        <div>\r\n            \r\n        </div>\r\n\r\n    </header>\r\n\r\n    <footer>\r\n\r\n\r\n    </footer>\r\n\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision d589912318bbf6b7c3029610b6fdab37a647a749)
+++ b/index.html	(date 1664890355002)
@@ -6,18 +6,10 @@
 </head>
 <body>
     <header>
-
-
         <div class="oui">
-            <h1>Oui il fait beau</h1>
-            <h2>Mais des fois il fait moche</h2>
-            <p>Creer une branche</p>
         </div>
-
         <div>
-            
         </div>
-
     </header>
 
     <footer>
